src/App/Tracker/Controller/Hooks/ReceiveCommentsHook.php

```diff
@@ -284,7 +299,20 @@ protected function updateComment($id)
 			$this->getContainer()->get('app')->close();
 		}
 
-		$this->triggerEvent('onCommentAfterUpdate', $table);
+		try
+		{
+			$issuetable = new IssuesTable($this->db);
+			$issuetable->load(array('issue_number' => $this->hookData->issue->number));
```

Comment:
This will not work as `issue_number` is not unique. You need to add the project ID here also:
```php
$issuetable->load(
	array(
		'issue_number' => $this->hookData->issue->number,
		'project_id' => $this->project->project_id
	)
);
```

That will be a unique combination (issue_number + project_id key is defined as unique for `#__issues` table).

Please use camelCase for vars: `$issuetable` -> `$issueTable`